test/contracts/core/tokens/rebalancingSetToken.spec.ts

```diff
@@ -620,7 +620,8 @@ contract('RebalancingSetToken', accounts => {
         const [minimumBid] = await rebalancingSetToken.getBiddingParameters.callAsync();
         await rebalanceAuctionModule.bid.sendTransactionAsync(
           rebalancingSetToken.address,
-          minimumBid
+          minimumBid,
+          false
```

Comment:
I noticed this when I was going in to update the specs to use the deployed contracts. Why don't you specify the transaction data when you send transactions?